### PR TITLE
Feature added. The mega barn.

### PR DESCRIPTION
**Pull request by CharlesTitzer**
This is horrible code. I'll allow it.

### CODE REVIEW

**Comment by RaazKH** [comment]
Ok let me see

File: Assets/Materials/New Material.mat [hunk 1]
```diff
@@ -78,6 +78,6 @@ Material:
     - _UVSec: 0
     - _ZWrite: 1
     m_Colors:
-    - _Color: {r: 0.3019608, g: 0.20465179, b: 0, a: 1}
+    - _Color: {r: 0.5188679, g: 0.3504043, b: 0, a: 1}
```
Comment (RaazKH) [inline Assets/Materials/New Material.mat:81]: The big barn is too red

**Review by RaazKH — CHANGES_REQUESTED** [review]
Change barn color

**Comment by CharlesTitzer** [comment]
My hands are shaking. Do you think the barn will make the final cut?

File: Assets/Materials/New Material.mat (hunk 1 above)
Comment (CharlesTitzer) [reply Assets/Materials/New Material.mat:81]: Wrong.

**Review by RaazKH — APPROVED** [review]
Very nice, approved 👍

**Pull request merged**